geode-management/src/main/java/org/apache/geode/management/api/ClusterManagementOperation.java

```diff
@@ -0,0 +1,36 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more contributor license
+ * agreements. See the NOTICE file distributed with this work for additional information regarding
+ * copyright ownership. The ASF licenses this file to You under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance with the License. You may obtain a
+ * copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package org.apache.geode.management.api;
+
+import org.apache.geode.lang.Identifiable;
+
+/**
+ *
+ * @param <V> the result type of the operation
```

Comment:
I think we should fill in the Javadoc for any User API classes. Second preference would be to delete any incomplete Javadocs if we think it's unnecessary to write Javadocs on a class.

Reply:
added javadocs, thanks for the reminder